packages/create-cloudflare/src/frameworks/versionMap.json

```diff
@@ -6,7 +6,7 @@
 	"hono": "0.2.6",
 	"next": "13.4.2",
 	"nuxt": "3.4.2",
-	"qwik": "1.1.x",
+	"qwik": "1.2.7",
```

Comment:
out of curiosity, since before it was `1.1.x` why not going to `1.2.x`?

Reply:
Didn't have a ton of time to test multiple 1.2.x versions so I just bumped to latest. I also didn't like that it was sort of a one off, I'd like to re-evaluate our version pinning and use `.x` for everything or pin to specific consistently.